business/iter8.go

```diff
@@ -0,0 +1,279 @@
+package business
+
+import (
+	"encoding/json"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kube "k8s.io/client-go/kubernetes"
+	"github.com/kiali/kiali/config"
+	"k8s.io/client-go/rest"
+	"time"
+	"fmt"
+	"github.com/kiali/kiali/log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	iter8v1alpha1 "github.com/iter8-tools/iter8-controller/pkg/apis/iter8/v1alpha1"
+	kialiConfig "github.com/kiali/kiali/config"
+)
+
+const GroupName = "iter8.tools"
```

Comment:
Note that these public variables are package scoped to whole "business" package.
So, to avoid clashes better to add a prefix i.e. "Iter8GroupName", "Iter8GroupVersion" I think.

Again, I'm just starting to look the code, probably I can comment later when I put all pieces together.

Reply:
Yes, will update to "Iter8GroupName" and "Iter8GroupVersion"